core/src/de/golfgl/gdxgamesvcs/IGameServiceClient.java

```diff
@@ -150,27 +161,32 @@
      * info level. If the connection is not open, this is no error - some game services allow submitting scores
      * without a user session.
      *
-     * @param leaderboardId
-     * @param score
+     * @param leaderboardId leaderboard id where the score is submitted
+     * @param score         score that will be submitted
      * @param tag           an optional information to post on the leader board, if API supports it. May be null.
      * @return false if submission couldn't be made. true if submit request was made (regardless if it was successful)
      */
     boolean submitToLeaderboard(String leaderboardId, long score, String tag);
 
     /**
-     * Fetches leader board entries
+     * Fetches leaderboard entries
      *
      * @param leaderBoardId   leaderboard to fetch
      * @param limit           limit how many entries to retrieve
      * @param relatedToPlayer only fetch scores around current player score or by current player (depending on Game
      *                        Service). If this is not possible, player-unrelated leaderboard entries are returned.
-     * @param callback
+     * @param callback        the listener that will be notified about the result
```

Comment:
To remain backwards compatibility, we should keep a method with the old parameters.

Reply:
Kept both versions of the method for backwards compatibility. If GameService supports timespans and collections the overloaded method can be used. If not, an exception is thrown.